modules/operserv/set_commitinterval.c

```diff
@@ -22,20 +22,43 @@ os_cmd_set_commitinterval_func(struct sourceinfo *const restrict si, const int p
 		return;
 	}
 
+	if (! db_save)
+	{
+		(void) command_fail(si, fault_badparams, _("You have no write-capable database backend loaded."));
```

Comment:
`fault_badparams` doesn't seem right here, though I'm not sure what the best choice would be tbh